src/theme/FlowStep.ts

```diff
@@ -0,0 +1,104 @@
+import { mode } from "@chakra-ui/theme-tools"
+
+export enum FlowStepStatus {
+  active = "ACTIVE",
+  inactive = "INACTIVE",
+  complete = "COMPLETE",
+}
+
+export enum FlowStepVariant {
+  vertical = "VERTICAL",
+  horizontal = "HORIZONTAL",
+}
+
+export enum FlowStepSize {
+  sm = "SM",
+  lg = "LG",
+}
```

Comment:
Sorry for changin my mind 😢  But i've noticed that we don't do the enums for the chakra related props in other components.

So my proposition would be to keep only `FlowStepStatus` as an enum, and revert variant and size to the type as it was before.

```suggestion
export enum FlowStepStatus {
  active = "ACTIVE",
  inactive = "INACTIVE",
  complete = "COMPLETE",
}

export type FlowStepVariant = "vertical" | "horizontal"
export type FlowStepSize = "sm" | "lg"
```

Reply:
Also **let's move those to the Flow Step component index file** so they are exportet from the lib. Right now we are not expotring them